packit/api.py

```diff
@@ -362,6 +390,161 @@ def update_source_git(self, revision_range: str):
                     f"Commit {commit} had no changes to be applied, skipping it."
                 )
 
+    def show_sync_status(
+        self, status: SynchronizationStatus, source_git: str, dist_git: str
+    ) -> None:
+        """Outputs the synchronization status of source-git and dist-git.
+
+        Args:
+            status: Synchronization status of source-git and dist-git.
+            source_git: Path to source-git as specified by the user.
+            dist_git: Path to dist-git as specified by the user.
+        """
+        if status.source_git_range_start and status.dist_git_range_start:
+            click.echo(f"'{source_git}' and '{dist_git}' have diverged.")
```

Comment:
Nice! I really like using `click.echo` here, instead of log. Makes more much sense and it's more user-friendly.